frontend/src/layouts/Platform.vue

```diff
@@ -103,15 +104,20 @@ export default {
                 this.dialog.disablePrimary = msg.disablePrimary
             }
             this.dialog.onConfirm = onConfirm
+            this.dialog.onCancel = onCancel
         },
-        clearDialog () {
+        clearDialog (cancelled) {
```

Comment:
`cancelled` always seems to be set to true, I'd suggest removing it and updating the card to only if onCancel is defined.

Reply:
> cancelled always seems to be set to true

That is correct however, `clearDialog(false)` should be called upon `onConfirm` (IMO) - so this parameter is present for "correctness".  The change to call `clearDialog in `onConfirm` is out of scope tho at this point.

Reply:
I think this falls under YAGNI and would suggest dropping it for now given it's not in use, but not going to block.